prov/verbs/src/verbs_info.c

```diff
@@ -962,7 +962,7 @@ int fi_ibv_getinfo(uint32_t version, const char *node, const char *service,
 	}
 
 	fi_ibv_destroy_ep(hints->ep_attr->type, rai, 
-		hints->ep_attr->type == FI_EP_RDM ? &(rdm_cm.listener) : &id);
+		FI_IBV_EP_TYPE_IS_RDM(hints) ? &(rdm_cm.listener) : &id);
```

Comment:
I had to add a 
if (hints && hints->ep_attr) 
before calling here otherwise I get a segfault when running fi_info